examples/calculator.js

```diff
@@ -16,7 +16,16 @@
 
 const assert = require('assert');
 const bacardi = require('bindings')('bacardi.node');
-const calculator = new bacardi.Calculator();
+const calculator = new bacardi.Calculator(0);
+
+describe('Test for constructor function', () => {
+  it('The constructor function print create time.', () => {
+    new bacardi.Calculator(0);
+    new bacardi.Calculator(1000);
+    new bacardi.Calculator(2000);
+    new bacardi.Calculator(3000);
+  });
```

Comment:
could you add more tests for exception case?
or it is fine to add exception case tests in next patches.

Reply:
Actually, I don't know how to use try-catch exception block in Javascript. 
But I will try this one. :)

Reply:
@hwanseung I updated it , please check +_+

Reply:
Fortunately, we have already other examples to test for try-catch exception.
You can refer to "Test for invalid arguments" test.